### PR TITLE
added remember me functionality to login screen

### PR DESCRIPTION
**Pull request by RaghavendranC-Eaton**
Fixes # added remember me functionality to login screen.

#### Changes proposed in this Pull Request:

-
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- yarn start:example2 or yarn start:example3
- Go to login page url: http://localhost:3000/login
- Enter email, password and select remember me checkbox then click "log in" button.
- In the homepage, click logout from the top right corner user menu option.
- Now check your login page that email and remember me pre-fill values.

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts.

**Comment by RaghavendranC-Eaton** [comment]
> Please resolve the conflicts.

@bkarambe Conflicts resolved. Pls check it.

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts

**Comment by joebochill** [comment]
To expand on Ekta's comments, we should be doing everything in Example3 now (which should be the only example at this point — pull latest from dev).

Most of this behavior should be happening in AuthUIActions.

On login, we should be calling the onUserAuthenticated function from the AppContextProvider (work with Manoj, as he's been removing the SecurityContextProvider and migrating its usage to the AppContextProvider instead) and passing through the user details including the rememberMe boolean value. If rememberMe is true, we should be storing the email value in local storage so we can recover it and prepopulate the login screen later.

In the initiateSecurity action, we should be looking in localstorage to see if rememberme is set to true and fetching the username/email. This then gets passed to the AppContextProvider so that the Login screen can read the values and display them on initial load.

File: login-workflow/example3/src/screens/Login.tsx [hunk 1]
```diff
@@ -1,53 +1,82 @@
...
-                onContactSupport={(): void => {
-                    // eslint-disable-next-line no-console
-                    console.log('onContactSupport');
-                    auth.navigate('contact-support');
-                }}
-                projectImage={<img src={EatonLogo} alt="logo" style={{ maxHeight: 80 }} />}
-                header={<DebugComponent />}
-            />
-        </>
+        <AuthContextProvider
```
[the remark this answers is not in the document]
Comment (RaghavendranC-Eaton) [reply login-workflow/example3/src/screens/Login.tsx]: @ektaghag-eaton I have removed the AuthContextProvider and updated in the AppRouter.

**Review by joebochill — APPROVED** [review]

**Pull request merged**